python/lsst/ip/isr/isrFunctions.py

```diff
@@ -665,6 +665,9 @@ def brighterFatterCorrection(exposure, kernel, maxIter, threshold, applyGain):
     applyGain : `Bool`
         If True, then the exposure values are scaled by the gain prior
         to correction.
+    gains : `dict` of `float`
```

Comment:
I think the recommendation for dict documentation has changed:
https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-dict

Reply:
Very nice, I didn't know that, thanks!